scripts/globals/items/recall_ring_jugner.lua

```diff
@@ -0,0 +1,24 @@
+-----------------------------------
+-- ID: 15841
+-- Recall ring: Jugner
+-- Enchantment: "Recall-Jugner"
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/keyitems")
+require("scripts/globals/teleports")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:hasKeyItem(xi.ki.JUGNER_GATE_CRYSTAL) == false then
+       result = 445
+    end
+    return result
+end
+
+item_object.onItemUse = function(target)
+    target:addStatusEffectEx(xi.effect.TELEPORT, 0, xi.teleport.id.JUGNER, 0, 1)
+end
+
+return item_object
```

Comment:
Could you please add a newline to the end of this file, and others that are also missing it?